Jenkinsfile.download_logs

```diff
@@ -0,0 +1,60 @@
+pipeline {
+    agent { label 'download_logs' }
+
+    parameters {
+        string(name: 'REMOTE_SERVICE_URL', defaultValue: 'https://api.openshift.com', description: 'Service URL')
+        string(name: 'ASSISTED_INSTALLER_DEPLOYMENT_BRANCH', defaultValue: 'master', description: 'Branch on assisted-installer-deployment repo to use')
+    }
+
+    triggers { cron('H/30 * * * *') }
+
+    environment {
+        SKIPPER_PARAMS = " "
+
+        LOGS_DEST = "/var/ai-logs/triage_logs"
+
+        // Credentials
+        OFFLINE_TOKEN = credentials('admin_offline_token')
+        JIRA_ACCESS_TOKEN = credentials('assisted-installer-bot-jira-access-token')
+        SLACK_TOKEN = credentials('slack-token')
+    }
+    options {
+      timeout(time: 90, unit: 'MINUTES')
+    }
+
+    stages {
+        stage('Init') {
+            steps {
+                sh "scripts/install_environment.sh install_skipper"
+                sh "make image_build"
+            }
+        }
+
+        stage('Cluster-logs') {
+            steps {
+                sh "make download_cluster_logs"
+            }
+        }
+
+        stage('Create tickets') {
+            steps {
+                sh "rm -rf assisted-installer-deployment"
+                sh "git clone https://github.com/openshift-assisted/assisted-installer-deployment --branch ${ASSISTED_INSTALLER_DEPLOYMENT_BRANCH}"
```

Comment:
You'll need to reverse that
Close test-infra and use assisted-installer-deployment

Reply:
Sorry **clone

Reply:
Correct, I did a clone of test-infra up in Init. In here I want to keep this clone to have an option to use job definition from master, but python scripts from a selected branch. What do you think?

Reply:
Sounds good